src/test/javascript/spec/helpers/mutable.ts

```diff
@@ -0,0 +1,3 @@
+export type Mutable<T> = {
```

Comment:
I struggled to understand this :sweat_smile: could you maybe provide some docs to what this type does?
Doesn't `readonly` indicate that it is immutable?

Reply:
Yes, this removes the readonly, therefore the **`-`**`readonly`.
I will add a documentation for this.